requirements-base.txt

```diff
@@ -20,49 +20,50 @@ googleapis-common-protos==1.52.0
 google-cloud-pubsub==2.2.0
 google-cloud-storage==1.35.0
 jsonschema==3.2.0
-lxml>=4.6.1,<4.7.0
+lxml==4.6.2
 maxminddb==1.5.4
-mistune>0.7,<0.9
-mmh3>=2.3.1,<2.4
+mistune==0.8.4
+mmh3==2.3.1
 parsimonious==0.8.0
-petname>=2.6,<2.7
-phonenumberslite>=8.11.0,<8.12.0
-Pillow>=8.0.0,<9.0.0  # Pillow 8.x supports Python 3.6 to 3.9
-progressbar2>=3.32,<3.33
-psycopg2-binary>=2.7.0,<2.9.0
-PyJWT>=1.5.0,<1.6.0
+petname==2.6
+phonenumberslite==8.11.3
+Pillow==8.1.0
+progressbar2==3.32.0
+psycopg2-binary==2.7.7
+PyJWT==1.5.3
 pyrsistent==0.17.3
 python-dateutil>=2.0.0,<3.0.0
-python-memcached>=1.53,<2.0.0
-python3-saml>=1.4.0,<1.5
-python-u2flib-server>=5.0.0,<6.0.0
-PyYAML>=5.3,<5.4
-qrcode>=6.1.0,<6.2.0
-rb>=1.8.0,<2.0.0
+python-memcached==1.59
+python-u2flib-server==5.0.0
+python3-saml==1.4.1
+PyYAML==5.3
+qrcode==6.1
+rb==1.8
 redis-py-cluster==2.1.0
 redis==3.3.11
 requests-oauthlib==1.2.0
-requests[security]>=2.20.0,<2.21.0
+requests[security]==2.20.1
 # [start] jsonschema format validators
 rfc3339-validator==0.1.2
 rfc3986-validator==0.1.1
 # [end] jsonschema format validators
-sentry-relay>=0.8.3,<0.9.0
+sentry-relay==0.8.3
 sentry-sdk>=0.20.0,<0.21.0
```

Comment:
I will deal with this on another occasion as I need to fix the `pip freeze` before & after logic on `getsentry`.